### ISSUE TITLE
Change the way we create namespaces

### ISSUE THREAD

**Issue by mssola** [SUSE/Portus#105]
It has been pointed out that the way we create namespaces can be a bit confusing. Right now the logic goes as follows: since a namespace belongs to a team (and a team may have multiple namespaces), namespaces for a specific team should be created inside the team page.

Now, this is confusing, because since we have the "Namespaces" tab, the first thing that comes to mind for a user is to create it from there. The idea is to add the form for creating namespaces also in the "Namespaces" tab. In there, we should specify the team too.

So, to sum it up: there should be two ways for creating namespaces, one from the team page (as it is now), and another from the "Namespaces" page.

**Comment by flavio** [comment]
Seems fair, what does @cyntss think about it?

**Comment by mssola** [comment]
That was @cyntss idea actually :)

**Comment by flavio** [comment]
great!
